project_tree_structure.txt

```diff
@@ -28,6 +28,7 @@
 │   │   ├── Hash Table Quadratic Probing.txt
 │   │   ├── HashTableLinearProbingC++.txt
 │   │   └── TwoSum.c++
+│   ├── Heuristic_Search.cpp
```

Comment:
You do not need to add this in README.md or project tree structure
That will be done automatically on merge.

Reply:
It automatically updates README.md OR project tree structure. My new PR has conflict issues. Kindly help me to resolve them.